hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/fsdataset/impl/FsVolumeImpl.java

```diff
@@ -1291,7 +1291,9 @@ public ReplicaInPipeline append(String bpid, ReplicaInfo replicaInfo,
 
     // rename meta file to rbw directory
     // rename block file to rbw directory
+    long oldReplicaLength = replicaInfo.getMetadataLength() + replicaInfo.getBlockDataLength();
```

Comment:
`long oldReplicaLength = replicaInfo.getNumBytes() + replicaInfo.getMetadataLength();`

Reply:
Thanks @ZanderXu for your reviews, I have edited.